### PR TITLE
UCP/RNDV-GET: fixed processing of message size

### LINKED ISSUE #2275: failure in dcx/test_ucp_tag_xfer.contig_unexp/1
Looks like the test is trying to use SW RNDV over get_zcopy with 1 byte, which is illegal
need to make sure rndv_thresh >= min_get_zcopy

http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6178/label=hpc-test-node3,worker=0/console
```
23:30:12 [ RUN      ] dc/test_ucp_tag_stats.rndv_unexpected/0
23:30:12 [       OK ] dc/test_ucp_tag_stats.rndv_unexpected/0 (296 ms)
23:30:12 [----------] 2 tests from dc/test_ucp_tag_stats (595 ms total)
23:30:12 
23:30:12 [----------] 92 tests from dcx/test_ucp_tag_xfer
23:30:12 [ RUN      ] dcx/test_ucp_tag_xfer.contig_unexp/1
23:30:12 [     INFO ] 0 50x10^1 50x10^2 50x10^3 50x10^4 15x10^5 [1518039016.496396] [hpc-test-node3:59619:0]        dc_mlx5.c:509  UCX  ERROR Invalid get_zcopy length: 1 (expected: >= 65)
```

### PR DESCRIPTION
**Pull request by hoopoepg**
- added evaluation for min zcopy value

fixes #2275
picked from #2279

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6192/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6195/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6206/ for details (Mellanox internal link).

File: test/gtest/ucp/test_ucp_tag_xfer.cc [hunk 1]
```diff
@@ -484,6 +486,54 @@ size_t test_ucp_tag_xfer::do_xfer(const void *sendbuf, void *recvbuf,
...
+    size_t offset;
+    size_t length;
+    ucs::detail::message_stream *ms;
+
+    skip_err_handling();
+
+    EXPECT_EQ(posix_memalign(&send_buf, 8192, buf_size), 0);
+    EXPECT_EQ(posix_memalign(&recv_buf, 8192, buf_size), 0);
+
+    for (offset = 0; offset <= max_offset; offset += offset_step) {
```
Comment (yosefe) [inline test/gtest/ucp/test_ucp_tag_xfer.cc]: maybe we need to skip on valgrind? how long does this test take?

Comment (hoopoepg) [reply test/gtest/ucp/test_ucp_tag_xfer.cc]: about 90 secs per transport
I will update it to skip on valgrind

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6238/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6239/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6240/ for details (Mellanox internal link).

**Pull request merged**